src/jaegertracing/UDPTransportTest.cpp

```diff
@@ -100,7 +100,7 @@ TEST(UDPTransport, testExceptions)
         MockUDPClient::ExceptionType::kString
     };
     for (auto type : exceptionTypes) {
-        MockUDPTransport sender(net::IPAddress(), 0, type);
+        MockUDPTransport sender(net::IPAddress::v4("localhost", 0), 0, type);
```

Comment:
Why isn't this the default constructor?

Reply:
because the behavior on windows is a bit different.
In `UDPClient::UDPClient`, `v_socket.connect(_serverAddr);` throws an exception: 
**Cannot connect socket to remote address { family=2, addr=0.0.0.0, port=0 }**